publish.sh

```diff
@@ -1,6 +1,6 @@
 #!/bin/sh -e
 
-cd site
+cd _site
```

Comment:
FYI: okay for now, I guess, but I expect we'll want to follow up by removing this script entirely.

Reply:
I think we should wait until post-merge before we obliterate the script.